webpack.sample.config.js

```diff
@@ -43,164 +43,168 @@ let localOwaFolder;
 let devtool;
 
 var getConfig = function () {
-	  var config;
-
-	  try {
-	    // look for config file
-	    config = require('./config.json');
-	  } catch (err) {
-	    // create file with defaults if not found
-	    config = {
-	      'LOCAL_OWA_FOLDER': '/Users/name/openmrs-standalone-2.4/appdata/owa/',
-	      'APP_ENTRY_POINT': 'http://localhost:8081/openmrs-standalone/owa/openmrs-owa-orderentry/index.html'
-	    };
-
-	    fs.writeFile('config.json', JSON.stringify(config));
-
-	  } finally {
-	    return config;
-	  };
-	}
+	var config;
+
+	try {
+		// look for config file
+		config = require('./config.json');
+	} catch (err) {
+		// create file with defaults if not found
+		config = {
+			'LOCAL_OWA_FOLDER': '/Users/name/openmrs-standalone-2.4/appdata/owa/',
+			'APP_ENTRY_POINT': 'http://localhost:8081/openmrs-standalone/owa/openmrs-owa-orderentry/index.html'
```

Comment:
Did you intentionally include openmrs-standalone in the url?

Reply:
Okay, this is a sample webpack config that should be used by anyone trying to clone this project. A detailed instruction on how to use this file (and modify) it is given in the readme section of the project. Thanks